catalyst/dl/utils/quantization.py

```diff
@@ -20,7 +21,8 @@
 
 logger = logging.getLogger(__name__)
 
-torch.backends.quantized.engine = "qnnpack"
+if not platform.system() == "Windows":
+    torch.backends.quantized.engine = "qnnpack"
```

Comment:
Do we _really_ need to change state of the quantization engine from the global context? Since it's already set from the save_quantized_model - perhaps we can remove it here?